osu.Game.Rulesets.Mania/UI/Components/StageHint.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics.Containers;
+using osu.Game.Graphics;
+using osuTK.Graphics;
+
+namespace osu.Game.Rulesets.Mania.UI.Components
+{
+    public abstract class StageHint : CompositeDrawable, IHasAccentColour
```

Comment:
not sure this base implementation need exist if it's this empty. if accent colour is the only usage, you can just null-cast is to pass in the colour:

```csharp
if (hitTarget is IhasAccentColour colouredHitTarget)
	colouredHitTarget.AccentColour = AccentColour;
```